MyLibrary/MyLibraryApp/MyLibrary.WebUI/Pages/Index.razor

```diff
@@ -0,0 +1,39 @@
+﻿@page "/"
+
+<PageTitle>Index</PageTitle>
+
+<Banner></Banner>
+
+
+
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
+<p></p>
```

Comment:
All these empty paragraphs should be removed as they only create confusion when reading the code.